lib/util/objects.js

```diff
@@ -0,0 +1,11 @@
+/*
+ * Copyright 2020 New Relic Corporation. All rights reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+'use strict'
+exports = module.exports = { isSimpleObject }
+
+function isSimpleObject(thing) {
```

Comment:
Missing JSDoc. 🤓

Reply:
Can you also write some unit tests for these new functions?